dsc/tests/dsc_config_get.tests.ps1

```diff
@@ -55,4 +55,26 @@ Describe 'dsc config get tests' {
         $result.metadata.'Microsoft.DSC'.securityContext | Should -Not -BeNullOrEmpty
         $LASTEXITCODE | Should -Be 0
     }
+
+    It 'json progress for config subcommand' {
+        $config_yaml = @"
+            `$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json
+            resources:
+            - name: Echo
+              type: Microsoft.DSC.Debug/Echo
+              properties:
+                output: hello
+"@
+        $config_yaml | dsc config get --output-format json -f - 2> $TestDrive/ErrorStream.txt
+        $LASTEXITCODE | Should -Be 0
+        $lines = Get-Content $TestDrive/ErrorStream.txt
+        $ProgressMessagesFound = $False
+        foreach ($line in $lines) {
```

Comment:
instead of comparing text, since all output should be JSON, you should convert every line from json and check the object